src/components/registerV2/backupPassphrase.js

```diff
@@ -1,65 +1,16 @@
-import React from 'react';
-import QRCode from 'qrcode.react';
-import { CopyToClipboard } from 'react-copy-to-clipboard';
 import { translate } from 'react-i18next';
-import moment from 'moment';
+import React from 'react';
 import grid from 'flexboxgrid/dist/flexboxgrid.css';
+
 import { PrimaryButtonV2, TertiaryButtonV2 } from '../toolbox/buttons/button';
-import links from '../../constants/externalLinks';
-import Tooltip from '../toolbox/tooltip/tooltip';
-import { InputV2 } from '../toolbox/inputsV2';
-import renderPaperwallet from '../../utils/paperwallet';
-import svgIcons from '../../utils/svgIcons';
+import PassphraseBackup from '../passphraseBackup';
 import registerStyles from './registerV2.css';
-import styles from './backupPassphrase.css';
 
 class BackupPassphrase extends React.Component {
```

Comment:
This component now looks like just a presentational component so maybe instead of a class you can use a stateless component function and then just make the destructuring in the parameters and not use the class.
I think looks better what do you think?